### ISSUE TITLE
asymmetric objects

### ISSUE THREAD

**Issue by unnamed333user** [TAO-TAO-TAO-TAO-TAO/SD-Net#1]
hi @TAO-TAO-TAO-TAO-TAO 
can this model work with asymmetric objects as well ?

**Comment by TAO-TAO-TAO-TAO-TAO** [comment]
Of course, our method significantly improves the prediction performance for symmetric objects and is also applicable to asymmetric objects

**Comment by unnamed333user** [comment]
tks @TAO-TAO-TAO-TAO-TAO 
Another question is that this model can be used with 1 class of object at a time, right ?

**Comment by TAO-TAO-TAO-TAO-TAO** [comment]
We are not a Category-Level pose estimation algorithm, but an instance level one

**Comment by unnamed333user** [comment]
tks @TAO-TAO-TAO-TAO-TAO  for your helpful response.

**Comment by unnamed333user** [comment]
the code is not released completely, is it ? @TAO-TAO-TAO-TAO-TAO

**Comment by TAO-TAO-TAO-TAO-TAO** [comment]
Hi, the work is still under review, I hope I'll be able to give you an answer soon.
Thanks for your interest.

> 代码没有完全发布，是吗？

**Comment by khanh0812** [comment]
Many thanks for your great work, could you send me released source code via email? @TAO-TAO-TAO-TAO-TAO
